pwnlib/elf/elf.py

```diff
@@ -2074,7 +2074,7 @@ def checksec(self, banner=True, color=True):
             "NX:".ljust(12) + {
                 True:  green("NX enabled"),
                 False: red("NX disabled"),
-                None: yellow("NX unknown - GNU_STACK missing"),
+                None: yellow("NX unknown"),
```

Comment:
Maybe this would get the point across?
```suggestion
                None:  yellow("NX enabled on new kernels"),
```

Reply:
Ya that seems much better. My biggest issue was the GNU_STACK_MISSING part which was incorrect in my particular case, if I remember the situation I was in at the time correctly. But I think this clears that up even more. 

Thanks for following up! I completely forgot about this issue.